Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -635,13 +619,49 @@ void CGPathAddRelativeArc(
     UNIMPLEMENTED();
 }
 
+static inline D2D_RECT_F __CGRectToD2D_F(CGRect rect) {
+    return {
```

Comment:
No.

Please move the one from CGContext into D2DWrappers and adjust its visibility appropriately.